content/home-2020-oct/card_11/en-US.md

```diff
@@ -0,0 +1,11 @@
+---
+# card 17
+title: "Our community participation guidelines"
+tag_label: "Community"
+desc: "We put people first and do our best to recognize, appreciate and respect the diversity of our global contributors"
+link_url: "https://www.mozilla.org/en-US/about/governance/policies/participation/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card"
```

Comment:
No UTMs on links to **www**.mozilla.org. Adding them when linking to the same website causes problems with analytics.